contrib/runners/remote_command_runner/runner.yaml

```diff
@@ -0,0 +1,80 @@
+- aliases:
+  - run-remote
```

Comment:
While we are at it, we should also get rid of the old aliases which have been deprecated for a while now.

Let's just make sure upgrade notes is updated and all the code in st2contrib as well (if there is any action which still uses old and deprecated runner name).